packages/flutter/lib/src/painting/box_shadow.dart

```diff
@@ -131,5 +131,5 @@ class BoxShadow extends ui.Shadow {
   int get hashCode => hashValues(color, offset, blurRadius, spreadRadius);
 
   @override
-  String toString() => 'BoxShadow($color, $offset, $blurRadius, $spreadRadius)';
+  String toString() => 'BoxShadow($color, $offset, ${debugFormatDouble(blurRadius)}, ${debugFormatDouble(spreadRadius)})';
```

Comment:
So we don't call `debug*` functions in release mode:

```dart
String description;
assert(() {
  description = 'BoxShadow($color, $offset, ${debugFormatDouble(blurRadius)}, ${debugFormatDouble(spreadRadius)})';
  return true;
}());
return description ?? super.toString();
```

Reply:
I think we generally consider toString to be a debug method. If we want to strip out debug code It should be done at a higher level